fairing/cloud/gcp.py

```diff
@@ -2,10 +2,14 @@
 from google.cloud import storage
 from fairing.constants import constants
 from kubernetes import client
+import logging
 
 import os
 import json
 
+logger = logging.getLogger(__name__)
+
+GCP_CREDS_SECRET_NAME = 'user-gcp-sa'
```

Comment:
This should go into fairing/constants/constants.py

Reply:
done.